aria/modeling/service_instance.py

```diff
@@ -1407,6 +1441,19 @@ def operation_template_fk(cls):
 
     # endregion
 
+    def configure(self):
```

Comment:
OK, so where? We don't have these separate module yet and I did not think I should be creating it as part of this PR.